http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -78,7 +78,7 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                                             referenceDatabaseNames: Set[String],
                                             workspaceId: UUID
   ): F[List[String]] = {
-    val wsmResourceDatabases = F.delay(
+    val wsmResourceDatabases = F.blocking(
```

Comment:
This was my bad -- all these WSM calls through the Java client are blocking so should all be wrapped in `F.blocking`, I think.